### PR TITLE
upgrade and bug fix.

### PR DESCRIPTION
**Pull request by Ackyo**

### CODE REVIEW

**Comment by guoyuepeng** [comment]
@Ackyo 

Thanks for your contributing.
Could you tell us what kind of bug you want to fix?

**Comment by guoyuepeng** [comment]
@Ackyo 

Could you tell us what is the background for this PR?

Thanks,

William

File: service/pom.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@ under the License.
         <eclipselink.version>2.6.0</eclipselink.version>
         <mysql.java.version>5.1.47</mysql.java.version>
         <postgresql.version>9.4.1212.jre7</postgresql.version>
-        <livy.core.version>0.3.0</livy.core.version>
+        <livy.core.version>0.7.1-incubating</livy.core.version>
```
Comment (guoyuepeng) [inline service/pom.xml:48]: why use this version in master version?

File: service/hibernate_mysql_pom.xml [hunk 2]
```diff
@@ -147,8 +147,8 @@ under the License.
         <!--livy-core-->
         <dependency>
             <groupId>com.cloudera.livy</groupId>
-            <artifactId>livy-core_2.10</artifactId>
-            <version>0.3.0</version>
+            <artifactId>livy-core_2.11</artifactId>
+            <version>0.7.1-incubating</version>
```
Comment (guoyuepeng) [inline service/hibernate_mysql_pom.xml:151]: what kind of issue you have encountered in version 0.3.0?

File: service/pom.xml [hunk 3]
```diff
@@ -32,8 +32,8 @@ under the License.
     <name>Apache Griffin :: Web Service</name>
 
     <properties>
-        <hadoop.version>2.7.1</hadoop.version>
-        <hive.version>2.2.0</hive.version>
+        <hadoop.version>3.0.0-cdh6.3.2</hadoop.version>
+        <hive.version>2.1.1-cdh6.3.2</hive.version>
```
Comment (wankunde) [inline service/pom.xml:36]: I think apache version is better?  @guoyuepeng @chitralverma  WDYT

Comment (chitralverma) [reply service/pom.xml:36]: I agree

**Pull request closed without merging**